### PR TITLE
Adding option to fail or pass the action status

### PR DESCRIPTION
**Pull request by sshniro**

### CODE REVIEW

**Comment by kingthorin** [comment]
To address the DCO requirement you'll need to sign-off the commit(s):

- https://github.com/zaproxy/zaproxy/blob/develop/CONTRIBUTING.md#developer-certificate-of-origin
- https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---signoff

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+## [Unreleased]
+### Added
+- Option to fail the status of the GitHub action if any alerts are found during the scan process.
+
+### Changed
+- Modified the action status to pass by default (previously fail by default)
+- Reduced docker logs
+
+## [0.1.0] - 2020-04-29
```
Comment (thc202) [inline CHANGELOG.md:14]: I think this was released later, `2020-05-15`?
The link should be added at the end of the file:
```
[0.1.0]: https://github.com/zaproxy/action-full-scan/compare/5842e3f84ec616724efb0230a6f6ab85146230c8...v0.1.0
```

Comment (sshniro) [reply CHANGELOG.md:14]: I took the release date, will update this.

**Comment by thc202** [comment]
Thank you!

**Pull request merged**